data/wanderer/wanderers start.txt

```diff
@@ -406,21 +337,39 @@ mission "Wanderers Conversation"
-			`	Rek is slightly larger than most of the Wanderers you have seen, with muted white and brown plumage and a calm, dignified bearing. "It is very [rare, exceptional] in our history that we have to [deal with, cope with] other species," he says, "and now we find ourselves dealing with two at once." He is speaking the Hai language, and the translation device does its best to render his words in your own language.`
+			`After you land, a group of Wanderers take you to the engineer from your last meeting. Explaining the device, the engineer says, "It will [function, succeed] on speech of typical [speed, tempo], but-" As they speak faster, the translation device produces gibberish. "This is Iktat Rek," they say, gesturing to another Wanderer. "He is [appointed, chosen] as the official [envoy, bearer] for human [visitors, couriers]."`
+			`	Rek is slightly larger than most of the Wanderers you have seen, with muted white and brown plumage and a calm, dignified bearing. "It is very [rare, exceptional] in our history that we have to [deal with, cope with] other species," he says, "and now we find ourselves dealing with two at once."`
 			choice
 				`	"How did you learn the Hai language?"`
 					goto language
 				`	"How long have you been fighting the Unfettered Hai?"`
-			
+					goto fighting
+				`	"Why were you able to create a translation device so quickly?"`
+
+			# Point of note: we have this technology now, in the real world. This is a race that can
+			# encode, in a few weeks, their race's entire self-hood in a way that a sentient AI can
+			# absorb. For them, making a translation machine for a simple meat-space language is trivial.
+			`	"[Automated, machine] language translation is a [primitive, early] application of machine [learning, intelligence]. For us it is [elementary, instinctual] mathematics. Our [thinkers, tinkerers] told me that most of the [work, enjoyment] was adapting the [interface, conceptualization, mechanism] to a human mind."`
+			choice
+				`	"How did you learn the Hai language?"`
+					goto language
+				`	"How long have you been fighting the Unfettered Hai?"`
+					goto fighting
+
+			label fighting
 			`	"It has been nearly twenty [years, cycles] now," he says. "At first only a single Hai ship [visited, invaded] us, but back then we had no defenses in place to stop it. For years the ship only [harvested, stole] food from our frontier worlds, and avoided combat. In our culture it is an [obligation, duty] to share our food with one who [asks for, demands] it, so we did not oppose the Hai ship. But then more Hai ships came, attacked one of our worlds and took it for themselves, and killed many of our people. We built warships and drove them back, and yet their [raids, attacks] still continue."`
 				goto food
 			
```

Comment:
label `language` may be worth changing. Since they have those algorithms, you could mention that some Wanderers have learned the Hai language with the help of those algorithms and devices.

Reply:
You need quite a substantial amount of data to train a language translation program. Also, every race in the galaxy that has had computers for more than a century or so will have these algorithms; there's nothing special about the Wanderers having them.